scripts/data/index.ts

```diff
@@ -0,0 +1,65 @@
+import * as dotenv from "dotenv";
+dotenv.config();
+import { writeFile as fsWriteFile } from "fs";
```

Comment:
Wouldn't rename Node.js functions, can be confusing.
Would either do:
```js
import { writeFile } from "node:fs";
```
Or to make it clear it's FS:
```js
import fs from "node:fs";
```

Reply:
Yeah, I'd rename the local writeFile. That's some inherited code, didn't change anything here